### DISCUSSION TITLE
【求助】

### DISCUSSION THREAD

**Discussion by Fuklos** [share121/inter-knot#1933] [其他]
为什么我不能换头像
![QQ20240813-204636](https://github.com/user-attachments/assets/5e51e045-8863-4cf6-bea3-1676f75122aa)

**Comment by Fuklos** [comment]
为什么上一个删掉了还显示啊，早知道不换标题了

**Comment by AnbulesPeo** [comment]
你需要一个小于1mb的常见格式的图片作头像

**Reply by Fuklos** [reply]
是小于1mb的

**Comment by L502567** [comment]
有人出教程了，去看看吧

**Comment by arez-17** [comment]
应该要等几分钟吧，我的就是

**Comment by qiang-di** [comment]
关掉加速器
